src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -111,4 +121,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     response.sendRedirect("/user-page.html?user=" + recipient);
   }
+
+  private void translateMessages(List<Message> messages, String targetLanguageCode) {
+
+    Translate translate = TranslateOptions.getDefaultInstance().getService();
+
+    for (Message message : messages) {
+      String originalText = message.getText();
+
+      Translation translation =
```

Comment:
nit: this formatting looks a little off...run the formatted on this file? ditto below.

Reply:
I ran the formatter and made arbitrary changes and it changed back to this. Is there anything that looks wrong in particular? I can check the style guide manually.